engine/events/magikarp.asm

```diff
@@ -108,36 +108,18 @@ CalcMagikarpLength:
 ;   de: wEnemyMonDVs
 ;   bc: wPlayerID
```

Comment:
```suggestion
;   de: Magikarp's DVs (MON_DVS / wEnemyMonDVs )
;   bc: Trainer ID (MON_ID / wPlayerID)
```
Those are only the inputs when called from `LoadEnemyMon` in [engine/battle/core.asm](https://github.com/pret/pokecrystal/blob/master/engine/battle/core.asm). When called from `CheckMagikarpLength`, the inputs are `MON_DVS` and `MON_ID`. (This distinction is important, as `MON_ID` is the Original Trainer's ID, not the player's.)

Reply:
I see. I'll address your suggestions when I get my laptop back.

Reply:
This would be `MON_OT_ID` now.